Hippo.Web/Controllers/OrderController.cs

```diff
@@ -289,7 +289,7 @@ public async Task<IActionResult> UpdateBilling([FromBody] OrderPostModel model)
 
             //TODO: Validation
             //Updating an existing order without changing the status.
-            var existingOrder = await _dbContext.Orders.Include(a => a.PrincipalInvestigator).Include(a => a.Cluster).Include(a => a.Billings).FirstAsync(a => a.Id == model.Id);
+            var existingOrder = await _dbContext.Orders.Include(a => a.PrincipalInvestigator.Owner).Include(a => a.Cluster).Include(a => a.Billings).FirstAsync(a => a.Id == model.Id);
```

Comment:
Perhaps rename the `a` variables in those `Include(...)` calls to `o`, since they're called on Order collections?